### PR TITLE
chore use utils tools to convert to double type

### PR DESCRIPTION
**Pull request by zyxxoo**

### CODE REVIEW

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/NeighborRankTraverser.java [hunk 1]
```diff
@@ -244,8 +244,8 @@ private Ranks contributeNewLayer(List<Adjacencies> adjacencies,
         public Step(HugeGraph g, Directions direction, List<String> labels,
                     long degree, long skipDegree, int top, int capacity) {
             E.checkArgument(top > 0 && top <= MAX_TOP,
-                            "The top of each layer must be in (0, %s]",
-                            MAX_TOP);
+                            "The top of each layer must be in (0, %s], that " +
+                            "current is %s", MAX_TOP, top);
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/NeighborRankTraverser.java:248]: "that current is %s" => "bug got %s"

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java [hunk 2]
```diff
@@ -122,7 +123,7 @@ public Traverser(Id sourceV, Directions dir, Id label, String weight,
                          long limit) {
             this.source = sourceV;
             this.sources = ImmutableSet.of(new NodeWithWeight(
-                           0D, new Node(sourceV, null)));
+                    0D, new Node(sourceV, null)));
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java:126]: keep the origin code

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java [hunk 3]
```diff
@@ -155,7 +156,8 @@ public void forward() {
 
                     double currentWeight = this.edgeWeight(edge);
                     double weight = currentWeight + node.weight();
-                    NodeWithWeight nw = new NodeWithWeight(weight, target, node);
+                    NodeWithWeight nw =
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java:159]: ditto

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java [hunk 4]
```diff
@@ -170,7 +172,7 @@ public void forward() {
             }
 
             Map<Id, NodeWithWeight> sorted = CollectionUtil.sortByValue(
-                                             this.findingNodes, true);
+                    this.findingNodes, true);
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java:175]: ditto

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java [hunk 5]
```diff
@@ -277,7 +272,8 @@ public int compareTo(NodeWithWeight other) {
         }
     }
 
-    public static class WeightedPaths extends LinkedHashMap<Id, NodeWithWeight> {
+    public static class WeightedPaths extends
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/SingleSourceShortestPathTraverser.java:275]: ditto

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**